packages/ffe-lists/less/regular-lists.less

```diff
@@ -18,6 +18,7 @@
 //     <li>Massevis av ferdige, testede komponenter</li>
 //     <li>Likt design som resten av SpareBank 1</li>
 //     <li>Høyere utviklingshastighet</li>
+//     <li class="ffe-check-list__item--cross">Null bugs</li>
```

Comment:
I miss the base class `ffe-check-list__item`. I know it doesn't do anything, but that's why it is called a modifier in BEM.

Reply:
yeah I agree, I'll implement what @ivarni suggested above